example/spec/parallel/cnmWorkflow/CnmWorkflowFromSqsSpec.js

```diff
@@ -185,7 +185,7 @@ describe('The Cloud Notification Mechanism SQS workflow', () => {
 
       // create SNS topic for cnm response
       const snsTopicName = timestampedName(`${config.stackName}_CnmSqsTestTopic`);
-      const { TopicArn } = await sns().send(new CreateTopicCommand({ Name: snsTopicName }));
+      const { TopicArn } = await sns().send(new CreateTopicCommand({ Name: snsTopicName, KmsMasterKeyId: 'alias/aws/sns' }));
```

Comment:
I'm ok with the answer to this being "No" _but_ is it worth wrapping this command in the SNS module so that we're not duplicating 'alias/aws/sns' everywhere? It's not a big deal since it only impacts tests but worth suggesting.

Reply:
I don't disagree that drying it up is a fine idea - I was fence-sitting on this in terms of balancing abstraction-that-hides-detail vs maintenance, and probably came down on the wrong side of an easy regex.     Will update.

Reply:
@paulpilone  how about 439e74d7406

Reply:
Looks great! I think that'll make any new/updated tests much less likely to _not_ be encrypted.